Header header 

int32[] startRingIndex
int32[] endRingIndex

int32[]  pointColInd
float32[] pointRange 

int64 imuAvailable
int64 odomAvailable

float32 imuRollInit
float32 imuPitchInit
float32 imuYawInit

float32 initialGuessX
float32 initialGuessY
float32 initialGuessZ
float32 initialGuessRoll
float32 initialGuessPitch
float32 initialGuessYaw

sensor_msgs/PointCloud2 cloud_deskewed

sensor_msgs/PointCloud2 key_frame_cloud
sensor_msgs/PointCloud2 key_frame_color
sensor_msgs/PointCloud2 key_frame_poses
sensor_msgs/PointCloud2 key_frame_map
